R/target_sda.R

```diff
@@ -298,11 +298,16 @@ compute_abcd_adjusted_scenario <- function(data, corporate_economy) {
     group_by(.data$sector, .data$scenario_source, .data$region) %>%
     filter(.data$year == min(.data$year, na.rm = TRUE)) %>%
     select(
-      .data$sector,
-      .data$scenario_source,
-      .data$region,
-      baseline_emission_factor = .data$emission_factor_corporate_economy
+      all_of(
+        c(
+          "sector",
+          "scenario_source",
+          "region",
+          "emission_factor_corporate_economy"
+        )
+      )
     ) %>%
+    rename(baseline_emission_factor = "emission_factor_corporate_economy") %>%
```

Comment:
apparently, you can still do renaming in `select()` without warpping in `all_of()`, which could eliminate L310, e.g. 
``` r
tibble(a = 1, b = 2, c = 3, d = 4) %>% 
  select(
    all_of(c("a", "b")),
    new_c = "c"
    )
#> # A tibble: 1 × 3
#>       a     b new_c
#>   <dbl> <dbl> <dbl>
#> 1     1     2     3
```

Reply:
yeah I noticed that too. I actually only separated them because I found it easier to read